src/main/java/racingcar/domain/RoundManager.java

```diff
@@ -0,0 +1,43 @@
+package racingcar.domain;
+
+import racingcar.dto.RacingCarDto;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RoundManager {
+    private final List<RacingCar> racingCars = new ArrayList<>();
+    private final AdvanceJudgement advanceJudgement;
+
+    public RoundManager(AdvanceJudgement advanceJudgement) {
+        this.advanceJudgement = advanceJudgement;
+    }
+
+    public void addRacingCar(RacingCar racingCar) {
+        racingCars.add(racingCar);
+    }
+
+    public List<String> getCurrentRound() {
+        List<String> roundResult = new ArrayList<>();
+        racingCars.stream().forEach(racingCar -> {
+            roundResult.add(racingCar.getDesc());
+        });
```

Comment:
```suggestion
        racingCars.forEach(racingCar -> {
            roundResult.add(racingCar.getDesc());
        });
```
이렇게 작성할 수도 있겠네요 ! (참고 : [Java 반복문](https://tecoble.techcourse.co.kr/post/2020-08-31-java-loop/))